manifests/l/LX-Systems/WinMute/2.1.2/LX-Systems.WinMute.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: LX-Systems.WinMute
+PackageVersion: 2.1.2

```

```suggestion
PackageVersion: 2.1.2.0
```
This must match Control Panel.

You'll also need to change `manifests/l/LX-Systems/WinMute/2.1.2` to `manifests/l/LX-Systems/WinMute/2.1.2.0`